UselessMine.cpp

```diff
@@ -323,6 +344,28 @@ void UselessMine::Event (bz_EventData *eventData)
                 else if (mine.defused)
                 {
                     dieData->killerID = mine.defuserID;
+
+                    if (playerID != mine.owner)
+                    {
+                        if (!defusalMessages.empty())
+                        {
+                            // The random number used to fetch a random taunting defusal message
+                            int randomNumber = rand() % defusalMessages.size();
+
+                            // Get a random defusal message
+                            std::string defusalMessage = defusalMessages.at(randomNumber);
+                            bz_sendTextMessage(BZ_SERVER, BZ_ALLUSERS, formatMineMessage(defusalMessage, bz_getPlayerCallsign(mine.defuserID), owner).c_str());
```

Comment:
```cpp
std::string formatMineMessage (std::string msg, std::string victim, std::string owner)
```

Wouldn't `bz_getPlayerCallsign(mine.defuserID)` be the owner of the mine here? And `owner` be the victim?